lib/model/query/entities.js

```diff
@@ -351,7 +378,7 @@ const _getFormDefActions = (oneFirst, datasetId, formDefId) => oneFirst(sql`
 // so any errors can be rolled back and logged as an entity processing error.
 const _processSubmissionEvent = (event, parentEvent) => async ({ Audits, Datasets, Entities, Submissions, Forms, oneFirst }) => {
   const { submissionId, submissionDefId } = event.details;
-  // TODO: check parentEvent details to determine if this is a forced reprocessing or not
+  const forceOutOfOrderProcessing = parentEvent?.details?.force === true;
```

Comment:
`forceOutOfOrderProcessing` could be a 3rd argument (defaulting to false) or something instead of being passed in within the `parentEvent`.

Reply:
ah, i decided against it because i'd have to pass it through `processSubmissionEvent` AND `_processSubmissionEvent`